server/app-engine/static/graas.js

```diff
@@ -471,18 +491,22 @@ function handleOkay() {
     p.style.background = "green";
     var p = document.getElementById(BUS_SELECT_DROPDOWN);
     vehicleID = p.value
+
+    document.cookie = `${VEHICLE_ID_COOKIE_NAME}=${vehicleID}; max-age=${MAX_VEHICLE_ID_AGE_SECS}`;
```

Comment:
Did you test that the cookie expires at max-age?

Reply:
Yep - tested on a MAX_VEHICLE_ID_AGE_SECS of 5 and confirmed it works as expected